mappings/net/minecraft/entity/ai/goal/TameableTargetGoal.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_1404 net/minecraft/entity/ai/goal/FollowTargetIfTamedGoal
+CLASS net/minecraft/class_1404 net/minecraft/entity/ai/goal/TameableTargetGoal
```

Comment:
![image](https://user-images.githubusercontent.com/7806504/124409435-e880ca80-dd0d-11eb-9d2e-9cc0b3ded9b8.png)
Should be `UntamedActiveTargetGoal`